docs/identity-service.md

```diff
@@ -3,6 +3,14 @@
 ## Get primary cloud identity for authenticated workload
 `GET /identities/identity`
```

Comment:
All of these need the API version in the querystring.

`?api-version=2020-09-01`